packages/block-library/src/post-tags/index.php

```diff
@@ -20,11 +20,17 @@ function render_block_core_post_tags( $attributes, $content, $block ) {
 
 	$post_tags = get_the_tags( $block->context['postId'] );
 	if ( ! empty( $post_tags ) ) {
-		$output = '';
+		$classes = 'wp-block-post-tags';
+		$output  = sprintf( '<div class="%1$s">', esc_attr( $classes ) );
```

Comment:
Given that we are declaring `$classes` above, we could just hardcode it within this string and avoid calling `sprintf` and `esc_attr`?

```suggestion
		$output  = '<div class="wp-block-post-tags">';
```

Reply:
@Copons and I discussed this earlier, but we were planning to add text alignment to the `post tags` block. https://github.com/WordPress/gutenberg/pull/24082#discussion_r462237222 where `sprintf` would make more sense.

Reply:
Sounds good then. 👍